src/lava/magma/runtime/message_infrastructure/examples/ros2/ros_talk_with_dds_py/ros_talk_with_dds_py/subscriber.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import rclpy
+from rclpy.node import Node
+from rclpy.qos import qos_profile_system_default
+
+from ddsmetadata.msg import DDSMetaData
+
+
+class MinimalSubscriber(Node):
+
+    def __init__(self):
+        super().__init__('minimal_subscriber')
+        ros_qos = qos_profile_system_default
+        self.subscription = self.create_subscription(
+            DDSMetaData,
+            'dds_topic',
+            self.listener_callback,
+            qos_profile_system_default)
+        self.subscription  # prevent unused variable warning
+
+    def listener_callback(self, msg):
```

Comment:
could add a function that converts DDSMetaData to numpy array into utils.py?

Reply:
Fix in commit [eca5172](https://github.com/lava-nc/lava/pull/498/commits/eca5172e5d3126b0600e0ec612a80fab0558cf70)